djedi/templatetags/staticfiles.py

```diff
@@ -0,0 +1,20 @@
+import django
+from django import template
+
+register = template.Library()
+
+if django.VERSION > (1, 9):
+    from django.templatetags.static import static as _static
+    from django.templatetags.static import do_static as _do_static
+else:
+    from django.contrib.templatetags.staticfiles import static as _static
+    from django.contrib.templatetags.staticfiles import do_static as _do_static
+
+
+def static(path):
+    return _static(path)
+
+
+@register.tag('static')
+def do_static(parser, token):
+    return _do_static(parser, token)
```

Comment:
Are these wrappers really needed? Wouldn't it be enough with the imports above, without the `as ...`?